Documentation/unit-files-and-scheduling.md

```diff
@@ -104,29 +98,74 @@ One must use the entire ID when setting `MachineID` - the shortened ID returned
 fleet depends on its host to generate an identifier at `/etc/machine-id`, which is handled today by systemd.
 Read more about machine IDs in the [official systemd documentation][machine-id].
 
-[machine-id]: http://www.freedesktop.org/software/systemd/man/machine-id.html
-
 ##### Schedule unit to machine with specific metadata
 
 The `MachineMetadata` option of a unit file allows you to set conditional metadata required for a machine to be elegible.
 
-```
+```ini
 [X-Fleet]
 MachineMetadata="region=us-east-1" "diskType=SSD"
 ```
 
-This requires an eligible machine to have at least the `region` and `diskType` keys set accordingly. A single key may also be defined multiple times, in which case only one of the conditions needs to be met:
+This requires an eligible machine to have at least the `region` and `diskType` keys set accordingly. This logic could be represented as follows:
 
+```sql
+region=us-east-1 AND diskType=SSD
 ```
+
+A single key may also be defined multiple times, in which case only one of the conditions needs to be met:
+
+```ini
 [X-Fleet]
 MachineMetadata=region=us-east-1
 MachineMetadata=region=us-west-1
 ```
 
-This would allow a machine to match just one of the provided values to be considered eligible to run.
+This would allow a machine to match just one of the provided values to be considered eligible to run. This logic could be represented as follows:
+
+```sql
+region=us-east-1 OR region=us-west-1
+```
+
+If we combine two previous examples in one:
+
+```ini
+[X-Fleet]
+MachineMetadata="region=us-east-1" "diskType=SSD"
+MachineMetadata=region=us-west-1
+```
+
+the logic would be as follows:
+
+```sql
+diskType=SSD AND (region=us-east-1 OR region=us-west-1)
+```
+
+Previous example schedules maximum one unit accross your cluster depending on first met requirement. If you add `Global=true`:
```

Comment:
The previous example schedules at most one unit across your cluster, depending on the first satisfied requirement.